app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    flash[:notice] = "Account successfully deleted!"
+    redirect_to root_path
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @email = params[:user][:email]
+    if @user.email != @email
+      if @user.update(email: @email)
+        flash[:notice] = "Email confirmation message has been sent to your current email"
+        redirect_to root_path
+      else
+        flash.now[:alert] = "Incorrect email was entered"
```

Comment:
Šeit parasti mes liekam augsta līmeņa paziņojumus - tipa `User did not update successfully!`

un specifiskos fieldu errorus mēs ģenerējam ar kkādu view partial palīdžibu izmantojot validācijas kļūdas -@user.errors.full_messages, piemēram.

Un tād var arī zem katra individuāla fielda norenderēt kļūdas .

Iedomājies ja tev būtu 10 lauki ar 10 dažādām iespējamām kļūdām - kā tev šeit strādātu tas flash now tieši?

Ar kkādu switch case būtu pagrūti + reizē jau var būt ari vairākas kļūdas.